src/api/customer/customer-validator.ts

```diff
@@ -34,6 +34,26 @@ const createCustomerValidator = () => {
     });
 };
 
+const getCustomersAllValidator = () => {
+    return celebrate({
+        [Segments.QUERY]: Joi.object().keys({
+            offset: Joi.string(),
```

Comment:
Poderia ter tratado no próprio Joi, desse campo ser um número inteiro, e já com um valor padrão caso o usuário não informasse.

```suggestion
            offset: Joi
                .number()
                .integer()
                .min(0)
                .default(0),
```